### ISSUE TITLE
[1.1.16] boost_last_run_duration in utilities.php not always numeric

### ISSUE THREAD

**Issue by paulgevers** [Cacti/cacti#910]
As noted in issue #846, I test the installation of cacti with a recursive web crawl. It looks like the test framework of Ubuntu is so fast, it can trigger the following which I didn't see before. I think (but my php is poor) that $boost_last_run_duration may not be initialized correctly.

```
2017/08/08 18:38:12 - ERROR PHP WARNING: A non-numeric value encountered in file: /usr/share/cacti/site/utilities.php  on line: 2141
2017/08/08 18:38:12 - CMDPHP PHP ERROR WARNING Backtrace: (/utilities.php: 87 boost_display_run_status)(/utilities.php: 2141 CactiErrorHandler)(/lib/functions.php: 4393 cacti_debug_backtrace)
```
Line 2141 in cacti 1.1.16 reads:
```
	print (($boost_last_run_duration > 60) ? __('%d minutes', (int)($boost_last_run_duration/60)) : '' ) . __('%d seconds', $boost_last_run_duration%60);
```

**Comment by cigamit** [comment]
Thanks Paul.  We are really grateful that you put together the Web Crawl, it helps with the QA side of things.  So, thanks.  Have this fixed in a jiffy.

**Comment by cigamit** [comment]
Okay, resolved.  Thanks again for all the help.  

We are getting things closed just as soon as we can.  But we are still addressing a few bugs like the Percona bug.  That's a biggie.  Need to validate if those templates cause issues and what caused the issue, of if it's the fact that they are not using our API, and instead doing raw inserts into the template and preset tables and therefore messing things up on 1.x.  The raw SQL updates do scare me.

I expect that in the next month, we will be again redirecting our focus to big picture issues again (aka 1.2).  Once we decided to release 1.x, we had to keep knocking out the bugs until there was good stability before attacking some of the "big" features.  Getting close to that now.
